src/dat.c

```diff
@@ -228,7 +229,7 @@ int eraDat(int iy, int im, int id, double fd, double *deltat )
    if (i < 0) return -5;
 
 /* Get the Delta(AT). */
-   da = changes[(i < 0) ? 0 : i].delat;
+   da = changes[i].delat;
```

Comment:
I looked at that but you had already put in the line 228 change that cause the function to return early if `i` is negative so why are we checking the variable again immediately afterwards. This way we have minimize the changes between ERFA and SOFA.